### PR TITLE
Muzima 825

### PR DESCRIPTION
**Pull request by saraheisa**

### CODE REVIEW

**Comment by saraheisa** [comment]
@mssavai @benamonya 
Please review the PR
Thanks

File: omod/src/main/webapp/resources/js/custom/app.js [hunk 1]
```diff
@@ -220,6 +221,35 @@ muzimaCoreModule.factory('FileUploadService', function ($http) {
     };
 });
 
+muzimaCoreModule.factory('HtmlFormEntryService', function ($http) {
+
+    var moduleState = function () {
+
+        return $http.get('htmlFormEntrymoduleStatus.form');
```
Comment (benamonya) [inline omod/src/main/webapp/resources/js/custom/app.js]: For consistency purposes consider naming this using camelCase. m Should be capital. Likewise htmlformentry.form should be in camelCase

File: omod/src/main/webapp/resources/js/custom/controllers/HtmlFormEntryController.js [hunk 2]
```diff
@@ -0,0 +1,244 @@
...
+                var serverData = response.data;
+                $scope.htmlForms = serverData.objects;
+                $scope.markConvertedForms();
+                $scope.totalItems = serverData.objects.length;
+            }).catch(function (error) {
+                showErrorMessage("There was an error connecting the server");
+                console.info(error);
+            });
+
+        // HtmlFormEntryService.getDiscriminatorTypes()
```
Comment (benamonya) [inline omod/src/main/webapp/resources/js/custom/controllers/HtmlFormEntryController.js]: Remove the commented out code

File: omod/src/main/webapp/resources/js/custom/controllers/HtmlFormEntryController.js [hunk 3]
```diff
@@ -0,0 +1,244 @@
...
+        $ctrl.convertedForm = res.data;
+        alertFunc(1, 'Conversion Succeeded');
+    }).catch(function (error) {
+        $ctrl.converting = false;
+        alertFunc(0, 'Conversion Failed');
+        $uibModalInstance.dismiss('cancel');
+    });
+
+    // prevent routing while converting
+
```
Comment (benamonya) [inline omod/src/main/webapp/resources/js/custom/controllers/HtmlFormEntryController.js]: Remove spaces to enable one know it is a comment for the function

File: omod/src/main/webapp/resources/partials/htmlFormEntry.html [hunk 4]
```diff
@@ -0,0 +1,278 @@
...
+                    </div>
+                    <div>
+                        <p>You have not installed <b>"HTMLFormEntry Module"</b> or it is NOT started. Once it is
+                            installed and started, and you have designed and saved some HTMLForms, those forms will be
+                            visible here.</p>
+                    </div>
+                </div>
+            </div>
+
+            <!-- <div class="row panel panel-default" ng-show="totalItems > 1">
```
Comment (benamonya) [inline omod/src/main/webapp/resources/partials/htmlFormEntry.html]: Remove commented out code

File: omod/src/main/webapp/resources/partials/htmlFormEntry.html [hunk 5]
```diff
@@ -0,0 +1,278 @@
...
+                                        <td>Discriminator</td>
+                                        <td>{{$ctrl.convertedForm.discriminator? $ctrl.convertedForm.discriminator : ''}}</td>
+                                        </tr>
+                                    </tbody>
+                                    </table>
+                                </div>
+                                <div class="form-review">
+                                    <div ng-show="$ctrl.converting" class="wait-convert"></div>
+                                    <uib-tabset active="active" ng-show="!$ctrl.converting">
+                                        <uib-tab index="0" heading="HTML Review" class="tab-review" style="width: 50%; text-align: center !important; color: #004f47 !important;">
```
Comment (benamonya) [inline omod/src/main/webapp/resources/partials/htmlFormEntry.html]: Review should be renamed to Preview

**Pull request merged**